Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the include granted scopes to determine if this authorization request should use
+        /// incremental authorization (https://developers.google.com/+/web/api/rest/oauth#incremental-auth).
+        /// If this is provided with the value <c>true</c>, and the authorization request is granted, the 

```

Comment:
Makes sense, more succinct. Done.

> I think that we can also remove the c tag, and just write "true" or "false" since it's a test property

I don't quite follow, but also done - not including the quotes I assume.